docs/spikes/admin-features.md

```diff
@@ -0,0 +1,61 @@
+# Admin Features
+
+figma link : https://www.figma.com/file/mLCjpDLPLJSdNliLQ0gz46/Untitled?node-id=0%3A1
+
+<br>
+
+### Switching to Admin mode
+- A button on the home page for switching to admin mode
+- Once clicked,  a message can be showed `Admin mode activated`
+- Also this button can be used to go back to user mode (if it is clicked during admin mode and the user can return to user mode), the colour of button is different in user mode and admin mode also the text changes
+
+
+<img src="https://user-images.githubusercontent.com/53532851/96143081-669f0600-0f20-11eb-8d98-e4286b6be4ae.png" alt ="registration" width="300"/>
+
+
+
+### Admin Dashboard
+- Once `Admin mode` is clicked ,Admin mode will be activated and  the admin dashboard will appear which shows the admin features
+
+ **FEATURES** 
+ - stats
+ - feedback
+ - admin list
+ - chat
+ - revoke
+ - Users 
+
+<img src="https://user-images.githubusercontent.com/53532851/96143159-761e4f00-0f20-11eb-8da5-0b07f0d58e8e.png"  width="300"/>
+
+### Stats
+- This shows the number of current users , mentors, mentees , completed relations and the number of ongoing relations ,, It also shows if the users are increaing or decreasing
+<img src ="https://user-images.githubusercontent.com/53532851/95678745-3b4cac00-0bec-11eb-9b5e-72cf80b622bc.png" width="300"/>
+
+### Feedback
+- This page shows the reviews , which the users gave 
+<img src ="https://user-images.githubusercontent.com/53532851/96143178-7a4a6c80-0f20-11eb-9c8f-c561116fac55.png" width="300"/>
+
+### Admin list
+- This shows the list of current admins
+<img src ="https://user-images.githubusercontent.com/53532851/95680993-0e53c580-0bfb-11eb-911e-82dd726b354a.png" width="300"/>
+
+### Chat
+- A chat window for admins to discuss about features or any other topic , It also provides updates about new admins or if any admin is removed
+<img src ="https://user-images.githubusercontent.com/53532851/95681060-65f23100-0bfb-11eb-93f7-2f36eaf7244e.png" width="300"/>
+
+### Revoke
+- A page to remove any admin
+<img src ="https://user-images.githubusercontent.com/53532851/95681310-b453ff80-0bfc-11eb-9fe7-98575c316437.png" width="700"/>
+
+### Users
+- This lets the admin to report any user or make them admin 
+<img src ="https://user-images.githubusercontent.com/53532851/95681398-3b08dc80-0bfd-11eb-8d76-669f24df098e.png" width="700"/>
+
+## `Users` can also be divided into two features `block` and `make admin`
+<img src ="https://user-images.githubusercontent.com/53532851/96144129-75d28380-0f21-11eb-8208-35340ee9fe13.png" />
+
+
+
+
+
+
```

Comment:
Why there are so many blank lines here??

Reply:
These are by mistake , no specific reason , should I remove them ??

Reply:
Yes, you have to. Also, go through docs once as I found unnecessary blank lines in between too